service/models.py

```diff
@@ -95,7 +97,13 @@ def deserialize(self, data):
             self.product_id = data["product_id"]
             self.condition = getattr(Condition, data["condition"])
             self.quantity = data["quantity"]
-            self.restock_level = data["restock_level"]
+            if isinstance(data["restock_level"], int):
```

Comment:
@walkccc not sure if I can check like this. We may have empty restock level.

Reply:
@ChingKai5566 I think we can't. If `data` doesn't have the field `restock_level`, `data["restock_level"]` will raise `KeyError: 'restock_level'`. Does that make sense to you?

Reply:
Fixed in the latest version. Thx!